servatrice/src/serversocketinterface.cpp

```diff
@@ -520,6 +521,20 @@ void ServerSocketInterface::deckDelDirHelper(int basePathId)
     sqlInterface->execSqlQuery(query);
 }
 
+void ServerSocketInterface::sendServerMessage(const QString userName, const QString message)
+{
+    ServerSocketInterface *user = static_cast<ServerSocketInterface *>(server->getUsers().value(userName));
+    if (user) {
```

Comment:
Personally, (and this is minor code style), I find it more readable to do along the lines of

```
if (!something)
    return;
lotsOfStuff()
```
instead of
```
if (something) {
    lotsOfStuff()
}
```